### DISCUSSION TITLE
After installation, the first user created logged in and encountered an error when uploading images:App\Http\Middleware\AdminUserStatus::handle:57 Admin user not found

### DISCUSSION THREAD

**Discussion by iihorse** [LycheeOrg/Lychee#2299] [Q&A]
Version：5.2.1 ，After installation, the first user created logged in and encountered an error when uploading images:App\Http\Middleware\AdminUserStatus::handle:57 Admin user not found，May I ask how to solve it?

**Comment by ildyria** [comment]
Use `php artisan lychee:create_user <username> <password> --may-administrate`

**Reply by iihorse** [reply]
I created a new administrator user using the method you mentioned, and after logging in with this user, another error occurred when uploading images: App \ Actions \ HoneyPot \ BasePipe: throwNotFound: 40 The route admin could not be found

**Reply by ildyria** [reply]
I don't know what, but there is something very wrong with your installation.
